### ISSUE TITLE
Bipedal edema

### ISSUE THREAD

**Issue by johnwjudkins** [obophenotype/human-phenotype-ontology#5821]
**Preferred term label:**
Bipedal edema

**Synonyms**
**Definition (free text, please give PubMed ID)**
A pedal edema that inheres in a human being and results in swelling in both feet.

**Parent term (use hpo.jax.org/app)**
Pedal edema HP:0010741

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
pneumonia

**Your nano-attribution (ORCID)**
0000-0001-6595-0902

Bipedal edema was mentioned in #3473; however, the term I'm proposing is for the specific case of swelling in _both_ feet. Thus the equivalent axiom could be modified from 'Pedal edema' to be: 'has part' some (edematous and ('inheres in' exactly 2 leg) and (has modifier some abnormal)). This term would distinguish pedal edema due to a clot in one leg from pedal edema due to something more general like respiratory illness.

**Comment by pnrobinson** [comment]
Thanks for the suggestion, we are adding this term. I have adjusted the definition to

A palpable swelling in both feet and ankles caused by palpable swelling caused by an increase in interstitial fluid volume (excess fulid).

**Comment by pnrobinson** [comment]
The component 'inheres in' exactly 2 leg requires a good bit of computation, and instead we can use this definition
```
'has part' some 
    (edematous
     and ('inheres in' some leg)
and ('has modifier' some bilateral)
     and ('has modifier' some abnormal))
```

**Comment by pnrobinson** [comment]
added
